xml/admin_cephx.xml

```diff
@@ -843,4 +843,73 @@ exported keyring for client.admin
    </variablelist>
   </sect2>
  </sect1>
+ <sect1 xml:id="permissions-cephx">
+   <title>Authorization Permissions (capabilities) for &cephx;</title>
+   <para>
+     &ceph; uses the term <literal>capabilities</literal> (caps) to describe
+     authorizing an authenticated user to exercise the functionality of the
+     monitors, OSDs and metadata servers.
+   </para>
+   <para>
+     Capability syntax follows the form:
+   </para>
+<screen>
+{daemon-type} '{cap-spec}[, {cap-spec} ...]'
+</screen>
+   <para>
+     For example, monitor capabilities include <literal>r</literal>,
```

Comment:
This is quite a complex few paragraphs. Might I suggest a `variablelist` instead?